src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import {
+  Layout,
+  ProtectedRoute,
+  SplashScreen,
+  Login,
+  Register,
+  Doctors,
+  DoctorDetails,
+  Appointments,
+  NewAppointment,
+} from './components';
+
+function App() {
```

Comment:
- [ ] Your frontend design implementation for the details page is functional and pretty, however, it is very different as compared to the required design:

Your implementation:

![image](https://github.com/LoviMadaan/Appointment-frontend/assets/57421823/2967b333-cea9-4db6-b230-04db6a5859ec)


The required design:

![image](https://github.com/LoviMadaan/Appointment-frontend/assets/57421823/f9bf73ec-e13d-444f-9480-c5390a79427c)

Kindly, add more styling so the implementation is more like the required design.  You can do it. 💪 

That way, we will meet this requirement and your chances in the final evaluation will be vastly improved. 👍

Reply:
@Meltrust dear hope you are doing good I will make the changes you requested for but for doctor details it does not make sense because in actual design we have too many things to show which is for project of four to six people but if I move the doctor on the left and data on the right it is only doctor name and doctor bio it will look ugly I hope you kindly understand that

Reply:
I consider what my partner is trying to explain here, is just to add more styling on the page according to the given design, if we talk about the page where a list of doctors appears, this could be shown as this:
<img width="1001" alt="image" src="https://github.com/LoviMadaan/Appointment-frontend/assets/20563252/8508c522-47a4-4261-a84e-fd172c0c0455">

This is a carousel with the functionality to add more than 3 options but only visible 3 votes simultaneously.

Now, let's consider the view when a motorcycle is clicked, then it's possible to observe the details for the motorcycle in the entire page, is it possible to do it when I click a doctor in your project? If the answer is no, then is required to create this page view in the project to be more according to the given design. I hope this has a clear explanation of what is required here, anyway if you have more questions let me know. 😄